pulp_2to3_migration/tests/functional/util.py

```diff
@@ -24,3 +26,25 @@ def monitor_task(tasks_api, task_href):
     else:
         print("The task did not finish successfully.")
     return task
+
+
+def get_psql_smash_cmd(sql_statement):
```

Comment:
Why return the command instead of executing immediately?

Reply:
Semi-related, why execute from the command line vs. something like ```psycopg2```?

https://www.psycopg.org/docs/usage.html

Reply:
> Why return the command instead of executing immediately?

I tried to avoid instantiating smash cli client every time, I guess it can be passed here as an argument to execute right away.
To be clear what smash cli does - it literally runs the command inside the container `docker exec -it pulp psql ...`


> Semi-related, why execute from the command line vs. something like `psycopg2`?
> 
> https://www.psycopg.org/docs/usage.html

The problem with programmatic approach is the following:
 - func tests are running on the travis machine
 - the db is local in our dev environment and it's in the container in our CI
 - in order to connect from the host to the db in a container, the db should listen not only on localhost (which is doable but a bit of a pain if it's for the tests sake only) + the db port should be exposed (easily doable but yet another customisation)
 - x9c4 convinced me that smash cli (even though I really dislike it :)) is a better approach because you can configure different transport and have pulp in a container or on a different machine and smash cli will either run `docker exec` or ssh into a machine.

Suggestions are welcome.